ogrmerge.py

```diff
@@ -0,0 +1,620 @@
+# -*- coding: utf-8 -*-
```

Comment:
can this be a dependency instead of added to the project?

Reply:
Sadly no. It is one of the 'helper scripts' included in a GDAL installation and is buried a couple directories deep. Including it is cleaner than the previous hackish and fragile way I was searching for the file based on the python module's location.